uPortal-rendering/src/main/java/org/apereo/portal/character/stream/events/PortletContentPlaceholderEventImpl.java

```diff
@@ -35,30 +33,6 @@ public CharacterEventTypes getEventType() {
         return CharacterEventTypes.PORTLET_CONTENT;
     }
 
-    @Override
-    public int hashCode() {
-        int h = hash;
-        if (h == 0) {
-            h = internalHashCode();
-            hash = h;
-        }
-        return h;
-    }
-
-    private int internalHashCode() {
-        final int prime = 31;
-        int result = 1;
-        result =
-                prime * result
-                        + ((this.getPortletWindowId() == null)
-                                ? 0
-                                : this.getPortletWindowId().hashCode());
-        result =
-                prime * result
-                        + ((this.getEventType() == null) ? 0 : this.getEventType().hashCode());
-        return result;
-    }
```

Comment:
What has this been replaced by?
Is this inherited? Is there a lombok annotations I'm not able to see in the diff? Something else?

Reply:
All such implementations have been pulled up in the parent class to remove code duplicates.
Since all such methods were common to the 4/5 inheritors, I have applied pulled up refactor of the method and variable.


I can explore how I can replace the single implementation of this method ( now in the parent class) through Lombok in the next iteration.